spot-client/src/common/remote-control/xmpp-connection.js

```diff
@@ -230,6 +241,21 @@ export default class XmppConnection extends Emitter {
                     if (errors.length) {
                         const error = errors[0].children[0].tagName;
 
+                        if (error === 'conflict') {
+                            if (!firstConflictTimestamp) {
+                                firstConflictTimestamp = Date.now();
+                            }
+
+                            const timeSinceFirstConflict = Date.now() - firstConflictTimestamp;
+
+                            if (timeSinceFirstConflict <= CONFLICT_TIMEOUT) {
+                                logger.warn('Retry MUC join on conflict error', { timeSinceFirstConflict });
+                                setTimeout(() => this._joinMuc(roomLock), 5000);
```

Comment:
Maybe we should clear this timeout on destroy, to be safe?

Reply:
yes, good point